raiden/tests/unit/test_matrix_transport.py

```diff
@@ -196,3 +197,26 @@ def test_invite_tiebreaker():
 
     assert my_place_or_yours(address, address1) == address
     assert my_place_or_yours(address1, address2) == address1
+
+
+@pytest.mark.parametrize("max_batch_size", [20])
```

Comment:
I would just put this into the function body when there is no need for `parametrize`'s features. But that in the realm of personal preference.